### DISCUSSION TITLE
FBW A32NX MCDU FPLN EMPTY PAGE

### DISCUSSION THREAD

**Discussion by karimoDev** [flybywiresim/aircraft#7217] [Q&A]
HI 
I USE FSMS V1.19.9 AND FBW A32NX V0.8.1 DEV VERSION
AND EATCH TIME I WANT TO ENTER A FLIGHT PLAN IN DE MCDU I GET A AN EMPTY PAGE 

![2693fa062623e612479101a5b8d213e216dc658f](https://user-images.githubusercontent.com/59665986/169266112-8e753d54-a9da-4f6d-9dba-f9496d068b58.jpeg)

CAN SOME ONE HELP ME PLZ

**Comment by ErikNL80** [comment]
Looks like a missing font?? Have you tried to uninstall and install?

**Reply by karimoDev** [reply]
YES BUT I STILL HAVE THE SAME PROBLEM

**Comment by winfuk** [comment]
Same here 


![Mcdu](https://user-images.githubusercontent.com/106463862/170855261-4240ea1c-7e12-4ab9-8ce7-e613bf6ba722.png)

**Comment by tracernz** [comment]
If either of you know how to use the Coherent debugger from the SDK, any debug output would be very helpful here.

**Reply by winfuk** [reply]
this is what you looking for 

![plan1](https://user-images.githubusercontent.com/106463862/170930719-817933ca-d10c-45a1-8c5d-043f3a92270f.png)

What should I look for

**Reply by winfuk** [reply]
or maybe this 
![plan 2](https://user-images.githubusercontent.com/106463862/170930978-f3e45256-d7b0-41af-aecb-fda066d0dd12.png)

**Reply by winfuk** [reply]
I hope this is what you looking for 
![plan3](https://user-images.githubusercontent.com/106463862/171208200-4aaa38af-a3af-4087-8b28-11f72bb6bbe8.png)

**Reply by tracernz** [reply]
FBW only supports the latest version of MSFS. ☠️

**Reply by 2hwk** [reply]
Please make sure you are on the latest MSFS version.
